authz/authz-tool/tool/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java

```diff
@@ -544,11 +546,6 @@ public void doSave(RunData data)
 	{
 		SessionState state = ((JetspeedRunData) data).getPortletSessionState(((JetspeedRunData) data).getJs_peid());
 
-		if (!"POST".equals(data.getRequest().getMethod())) {
```

Comment:
It's not in your Oxford instance, as far as I can tell. One of your commits removed it, so I left it out.